### DISCUSSION TITLE
Possible to use dbaclone with Azure SQL sources?

### DISCUSSION THREAD

**Discussion by danstis** [sqlcollaborative/dbaclone#158] [Q&A] [answered]
Hi all,
Is it possible to use dbaclone with source databases hosted in Azure SQL?

I am looking to clone a production SQL DB that is hosted in Azure SQL to a local development copy in a local SQL server.
Currently it seems that the source server must be available via WinRM to use dbaclone.

Am I missing something obvious here?

Thanks, 
Dan

**Comment by sanderstad** [comment, marked as answer]
Hi Dan,

Cloning data from an Azure SQL DB does come with some extra challenges.

The fact is that with Azure SQL DB you don't have any local backups, or are able to run a backup-restore to create the image.
How dbaclone creates the images is by backing up (or using an existing backup) and restoring that backup inside a VHD file.

The only way I think that this could work with Azure SQL DB, is when you export the data from your Azure SQL DB to a BACPAC, deploy that BACPAC to create the database locally and then use that local copy to create the images and eventually the clones.

**Reply by danstis** [reply]
Thanks for the info, this is kind of where I have landed so far.

**Reply by MatthewBeste** [reply]
Hi I'm digging into this same problem right now. My question is the other way around. Assuming I have the local image all setup can I deploy it to azure sql?
